skeleton/extension/composer.json

```diff
@@ -1,59 +1,59 @@
 {
-  "name": "__VENDOR_NAME__/__PACKAGE_NAME__",
-  "license": "GPL-2.0-only",
-  "type": "ibexa-bundle",
-  "keywords": [
-    "ibexa-dxp"
-  ],
-  "require": {
-    "php": "^7.4 || ^8.0",
-    "ibexa/core": "^4.5",
-    "symfony/config": "^5.4",
-    "symfony/dependency-injection": "^5.4",
-    "symfony/event-dispatcher": "^5.4",
-    "symfony/http-foundation": "^5.4",
-    "symfony/http-kernel": "^5.4",
-    "symfony/yaml": "^5.4"
-  },
-  "require-dev": {
-    "phpunit/phpunit": "^9.0",
-    "ibexa/code-style": "^1.1",
-    "phpstan/phpstan": "^1.10",
-    "phpstan/phpstan-phpunit": "^1.3",
-    "phpstan/phpstan-symfony": "^1.3",
-    "qossmic/deptrac-shim": "^0.24.0 || ^1.0.2"
-  },
-  "autoload": {
-    "psr-4": {
-      "__VENDOR_NAMESPACE__\\Bundle\\__BUNDLE_NAME__\\": "src/bundle/",
-      "__VENDOR_NAMESPACE__\\Contracts\\__BUNDLE_NAME__\\": "src/contracts/",
-      "__VENDOR_NAMESPACE__\\__BUNDLE_NAME__\\": "src/lib/"
+    "name": "__VENDOR_NAME__/__PACKAGE_NAME__",
+    "license": "GPL-2.0-only",
+    "type": "ibexa-bundle",
+    "keywords": [
+        "ibexa-dxp"
+    ],
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "ibexa/core": "^4.5",
+        "symfony/config": "^5.4",
+        "symfony/dependency-injection": "^5.4",
+        "symfony/event-dispatcher": "^5.4",
+        "symfony/http-foundation": "^5.4",
+        "symfony/http-kernel": "^5.4",
+        "symfony/yaml": "^5.4"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.0",
```

Comment:
Sort packages, and add `sort-packages` under `config` as `true`, and I'll be a happy man.